### PR TITLE
fix: resource preset validation

### LINKED ISSUE #672: Fix resource preset dialog validation
Let's fix those minor input validation checks.
- prevent resource-preset creation from invalid input. (e.g. name, resource_slot values, etc.)
- make input available in according to minimum steps in each resources (e.g. shared memory can be assigned in increments of 0.01)

### PR DESCRIPTION
**Pull request by lizable**
This PR Resolves #672.

### CODE REVIEW

File: src/components/backend-ai-resource-preset-list.ts [hunk 1]
```diff
@@ -325,7 +331,7 @@ class BackendAiResourcePresetList extends BackendAIPage {
           </wl-button>
         </div>
       </backend-ai-dialog>
-      <backend-ai-dialog id="delete-resource-preset-dialog" fixed backdrop blockscrolling>
+      <backend-ai-dialog id="delete-resource-preset-dialog" fixed backdrop blqockscrolling>
```
Comment (adrysn) [inline src/components/backend-ai-resource-preset-list.ts]: Is attribute `blqockscrolling` exists, or just typo?

**Review by adrysn — CHANGES_REQUESTED** [review]
I cannot create/update a resource preset when the value of fGPU is a decimal. For example, when you input `0.2` to fGPU, you cannot update it.

image

File: src/components/backend-ai-resource-preset-list.ts (hunk 1 above)
Comment (inureyes) [reply src/components/backend-ai-resource-preset-list.ts]: @adrysn  It is definitely typo. haha

**Review by inureyes — APPROVED** [review]
LGTM

**Pull request merged**